lib/commanded/event/handler.ex

```diff
@@ -868,6 +872,35 @@ defmodule Commanded.Event.Handler do
     {:noreply, state}
   end
 
+  def extract_error_handler_module(opts) do
+    case Keyword.get(opts, :retry, :stop) do
+      :stop ->
+        Commanded.Event.ErrorHandler.Stop
+
+      :backoff ->
+        Commanded.Event.ErrorHandler.Backoff
+
+      :grumpy_old_bastard ->
+        Commanded.Event.ErrorHandler.GrumpyOldBastard
+
+      module when is_atom(module) ->
+        # todo check this at macro time, why wait?
+        if function_exported?(module, :handle_error, 3) do
+          module
+        else
```

Comment:
let it crash, HARD! I dont know, defensive programming drives me nuts

Reply:
I agree, this will crash at compile time